compiler-core/src/erlang/tests/snapshots/gleam_core__erlang__tests__strings__assert_const_concat_many_strings.snap

```diff
@@ -0,0 +1,37 @@
+---
+source: compiler-core/src/erlang/tests/strings.rs
+expression: "\nconst big_concat = \"a\" <> \"b\" <> \"c\" <> \"d\" <> \"e\" <> \"f\" <> \"g\" <> \"h\" <> \"i\" <> \"j\" <> \"k\" <> \"l\" <> \"m\" <> \"n\" <> \"o\" <> \"p\" <> \"q\" <> \"r\" <> \"s\" <> \"t\" <> \"u\" <> \"v\" <> \"w\" <> \"x\" <> \"y\" <> \"z\"\n\npub fn main() {\n  big_concat\n}\n"
+---
+-module(my@mod).
+-compile([no_auto_import, nowarn_unused_vars, nowarn_unused_function, nowarn_nomatch]).
+
+-export([main/0]).
+
+-spec main() -> binary().
+main() ->
+    <<"a"/utf8,
```

Comment:
This should render like this:

```erlang
main() ->
    <<"a"/utf8,
      "b"/utf8,
      "c"/utf8,
      ...
```

Reply:
ooh right, it's not just format problem, we don't need to `/binary` the concatenation, we can build a bit array of `/utf8` strings. This is a lot nicer now, I am starting to understand what you meant with "string concatenation doesn't require compile time code execution"